sections/sprint-5/usability-test-may-18th.tex

```diff
@@ -1,11 +1,24 @@
-\section{Usability test}\label{sec:evaluatin_test}
-The second test of the system and all of its components took place on the 18th of May.
-This test was conducted as the previous test had not managed to explore the functionality of the game as described in \autoref{sec:initial-test}.
-Due to the COVID-19 pandemic, the usability test was conducted using the group members as testers, and only one person from outside the group.
+\section{Evaluation on May 18th}\label{sec:evaluatin_test}
+The evaluation of the system and all of its components took place on the 18th of May.
+This was conducted as the previous test had not managed to explore the functionality of the game as described in \autoref{sec:initial-test}.
+In the time between the two tests, the majority of the discovered bugs were fixed.
+Due to the COVID-19 pandemic, the test was conducted using the group members as testers, and only one person from outside the group.
+It would have been preferable to get external users to conduct the system, but this was obvious not a viable solution due to the circumstances.
```

Comment:
```suggestion
It would have been preferable to get external users to participate in the test, but this was obviously not a viable solution due to the circumstances.
```